crontab/CleanUploadsTrash.inc

```diff
@@ -16,42 +17,40 @@ class CleanUploadsTrash extends BackgroundJob
         }
 
         // remove files from TRASH subdirectory older than 30 days
-        $cmd = join(" ", [
+        $process = new Process([
             "/usr/bin/find",
-            escapeshellarg($trash_dir),
-            "-type f",
-            "-mtime +30",
+            "$trash_dir",
+            "-type", "f",
+            "-mtime", "+30",
             "-delete",
         ]);
-        $output = $return = null;
-        exec($cmd, $output, $return);
-        if ($return != 0) {
+        $process->run();
+        $output = $process->getOutput();
+        $exitcode = $process->getExitCode();
```

Comment:
Nit: We should move `$exitcode` into the `$process->isSuccessful()` branch as it is only needed there.